docs/snippets/GPU_RemoteTensor_creation.cpp

```diff
@@ -0,0 +1,78 @@
+#include <openvino/runtime/core.hpp>
+#include <openvino/runtime/intel_gpu/ocl/va.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+
+void* allocate_usm_buffer(size_t size);
+cl_mem allocate_cl_mem(size_t size);
+cl::Buffer allocate_buffer(size_t size);
+cl::Image2D allocate_image(size_t size);
+
+int main() {
+    using namespace ov;
```

Comment:
do we need to use this? In this case `intel_gpu::ocl::ClContext` is not clickable in docs..

Reply:
removed